packages/opentelemetry/src/spanExporter.ts

```diff
@@ -239,14 +239,14 @@ export function createTransactionForOtelSpan(span: ReadableSpan): TransactionEve
 
   const sampleRate = span.attributes[SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE] as number | undefined;
 
-  const attributes: SpanAttributes = dropUndefinedKeys({
+  const attributes: SpanAttributes = {
```

Comment:
this and the removed invocation below was unnecessary, because we call `dropUndefinedKeys` on the whole transaction event further down, which recursively does this anyhow.